critiquebrainz/frontend/static/scripts/rating.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
```

Comment:
It's more code we have to test (the modifications we make) and maintain (we don't have a lot of resources).

If any bug fixes are made upstream, we have to manually merge them into our fork, instead of just bumping the version in package.json.

It doesn't look like you've made that many modifications: https://gist.github.com/mwiencek/838e5abd016ce6627c23ba47befe2d2d

You don't need to fork the code to modify `DEFAULTS`. You can mutate it externally through `$.fn.rating.Constructor.DEFAULTS`.

For the rest, why not just submit a patch upstream to support what you're doing? That way other people benefit, and we don't have to maintain our own fork.